arch/x86_64/Kconfig

```
@@ -49,6 +49,23 @@ config ARCH_CHIP_X86_64_CUSTOM
 
 endchoice
 
+if SCHED_BACKTRACE
+choice
+	prompt "Choose x86_64 unwinder"
+	default X86_64_UNWINDER_FRAME_POINTER
+	---help---
+		This determines which method will be used for unwinding nuttx stack
+		traces for debug.
+config X86_64_UNWINDER_FRAME_POINTER
```

```suggestion
		traces for debug.

config X86_64_UNWINDER_FRAME_POINTER
```
sorry, I didn't notice this before